### PR TITLE
Reduces padding on highlight for drop down

### PR DESCRIPTION
**Pull request by karmatosed**
This is a small change that fixes the drop-down spacing on highlight for the navigation block. It relates to suggestions in #26571

**Before**

image

**After**

image

### CODE REVIEW

**Comment by jasmussen** [comment]
Confirmed, this fixes the issue:

Screenshot 2020-10-30 at 08 26 40

However, my golly that dropdown needs a lot of love. Why is the text gray? Why is it allcaps? (I know @aaronrobertshaw is struggling with this exact issue in https://github.com/WordPress/gutenberg/pull/26444) 

Why is the popover borderless when not in focus?

Screenshot 2020-10-30 at 08 26 18

To be quite clear, none of this is your doing, and I really appreciate your taking a stab at improving this menu, it's clearly full of low hanging fruit. Here's the CSS as best I can tell:

Screenshot 2020-10-30 at 08 27 16

It seems to me the problem here is that panel titles are lumped in with the custom select control, when those should be separate. So I'd extract `.components-custom-select-control__menu li` and give it its own separate definition. Want to take a stab at that?

File: packages/block-editor/src/components/inserter/style.scss [hunk 1]
```diff
@@ -166,15 +166,18 @@ $block-inserter-tabs-height: 44px;
...
-.components-custom-select-control__menu li {
+.block-editor-inserter__panel-title button {
 	margin: 0 $grid-unit-15 0 0;
 	color: $gray-700;
 	text-transform: uppercase;
 	font-size: 11px;
 	font-weight: 500;
 }
 
+.components-custom-select-control__menu li.is-highlighted {
```
Comment (jasmussen) [inline packages/block-editor/src/components/inserter/style.scss:177]: Alright, I rebased this and removed the line that causes inheritance of the allcaps text in the custom dropdowns. That actually fixes the margin issue, so you can remove this line.

**Comment by jasmussen** [comment]
At some point the `package-lock.json` file was included, it shouldn't be. I can't recall exactly how to remove a changed file commit from a PR, but we need to remove that and then this one is good to ship.

**Comment by karmatosed** [comment]
@jasmussen Ah goodness that was the last commit, I'll work out a way to solve that and get it ready. Thanks.

**Comment by karmatosed** [comment]
@tellthemachines if this could get a final code review I think it might be able to slip in the group for the release. It's low priority but as a block already released would fix a bug. I would love to know if I've done the removal of the file correctly and if not learn how to do that though.

**Comment by tellthemachines** [comment]
@karmatosed the Navigation Block isn't in Core yet, so this doesn't need to go into 5.6, but happy to help out with it anyway! 

The `package-lock.json` shouldn't be deleted, just removed from the changeset because we're not interested in making any changes to it in this branch. You can do that locally in terminal with `git checkout master package-lock.json` (you have to be in the branch you made these changes in) and commit the result. That should get rid of the merge conflict too 😅

**Comment by karmatosed** [comment]
Thanks so much @tellthemachines, I've done that 🙏🏻

**Review by jasmussen — APPROVED** [review]
Before:

Screenshot 2020-11-12 at 13 16 41

After:

Screenshot 2020-11-12 at 13 16 51

Screenshot 2020-11-12 at 13 16 55

There's so much work to still be done on that dropdown menu, but this is a great step in the right direction.

**Comment by karmatosed** [comment]
Awesome, thanks. Let's get this PR in then and work can continue.

**Comment by ellatrix** [comment]
Why was `package-lock.json` updated? Tests are failing now in master.

**Comment by karmatosed** [comment]
@ellatrix regarding this comment:

> The package-lock.json shouldn't be deleted, just removed from the changeset because we're not interested in making any changes to it in this branch. You can do that locally in terminal with git checkout master package-lock.json (you have to be in the branch you made these changes in) and commit the result. That should get rid of the merge conflict too

Happy to revert this merge if issue is problem though.

**Comment by ellatrix** [comment]
Right, it should have been removed from the branch and synced with master?

**Comment by ellatrix** [comment]
And no worries, it's easy enough to fix. It's been partially reverted to make the tests happy. Not sure if the rest should be reverted, but it's not causing any failures.

**Pull request merged**